GliaWidgets/Public/Glia/Glia+StartEngagement.swift

```diff
@@ -24,40 +24,38 @@ extension Glia {
         in queueIds: [String] = [],
         sceneProvider: SceneProvider? = nil
     ) throws {
+        guard let interactor else {
```

Comment:
The first step on startEngagement call should be configuration check, which you moved into `getEngagementParameters`. If we keep as is, then integrator will not receive any exception if the SDK is not configured.
I'd propose to move `guard let interactor else` check inside `getEngagementParameters` method right below configuration check.

```swift
guard let configuration = self.configuration else { throw GliaError.sdkIsNotConfigured }

guard let interactor else {
   loggerPhase.logger.prefixed(Self.self).warning("Interactor is missing")
   throw GliaError.sdkIsNotConfigured
}

Reply:
Totally agree.

Fixed https://github.com/salemove/ios-sdk-widgets/commit/35b99fb60b80b3827f49db315f47b1c344efdea9